src/Workspaces/Core/MSBuild.BuildHost/Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj

```diff
@@ -34,6 +34,7 @@
     <PackageReference Include="System.CommandLine" />
     <PackageReference Include="System.Collections.Immutable" />
     <PackageReference Include="Newtonsoft.Json" />
+    <PackageReference Include="Microsoft.IO.Redist" Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'" />
```

Comment:
Womp

Reply:
Yep.

Reply:
It's busted you have to do this, you want all of MSBuild to be Exclude="Runtime"

I wonder if instead of referencing the packages for MSBuild which bring in it's entire NuGet graph, you could just reference the API it exposes.  As reference assemblies.  cc @baronfel